src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -47,13 +48,20 @@
     create_hub_bucket_if_it_does_not_exist,
     generate_default_hub_bucket_name,
     create_s3_object_reference_from_uri,
+    tag_hub_content,
+    get_jumpstart_model_and_version,
+    find_jumpstart_tags_for_hub_content,
+    summary_list_from_list_api_response,
 )
 from sagemaker.jumpstart.curated_hub.types import (
     HubContentDocument_v2,
     JumpStartModelInfo,
     S3ObjectLocation,
+    CuratedHubTag,
+    HubContentSummary,
 )
 
+LIST_HUB_CACHE = None
```

Comment:
This isn't a constant, so can we make it lower case? Also, the list hubs cache should be scoped to a `CuratedHub` and live within the class definition